### ISSUE TITLE
Build fails: Test case with interrupted SLEEP fails with MariaDB

### ISSUE THREAD

**Issue by zuiderkwast** [mysql-otp/mysql-otp#171]
See the builds for PR #170. Has the behavior changed in some MariaDB version?

We should figure out what's happening and probably let the test case accept the new behavior, with a comment, maybe conditionally depending on the DB version.

**Comment by zuiderkwast** [comment]
https://travis-ci.org/github/mysql-otp/mysql-otp/jobs/747463786:
```
    mysql_tests: timeout_test_...*failed*
in function mysql_tests:'-timeout_test_/0-fun-2-'/1 (test/mysql_tests.erl, line 1000)
in call from mysql_tests:'-timeout_test_/0-fun-6-'/1 (test/mysql_tests.erl, line 999)
**error:{assertEqual,[{module,mysql_tests},
              {line,1000},
              {expression,"mysql : query ( Pid , << \"SELECT SLEEP(5)\" >> , 40 )"},
              {expected,{ok,[<<"SLEEP(5)">>],[[1]]}},
              {value,{error,{1317,<<"70100">>,
                             <<"Query execution was "...>>}}}]}
```

**Comment by juhlig** [comment]
Not sure what might be the problem, it may be related to Travis in some way. In fact, I saw this PR #170 yesterday evening already, but the tests didn't even start at travis (though they had been queued), I only saw most of them had passed this morning, with two others still queued. @Maria-12648430 and me also had (different) problems with travis integration in one of our projects earlier, but didn't have the time to investigate further. In general, it seems that many projects (among them OTP) are currently moving away from travis in favor of github actions, and I think we should consider this step, also.
